contrib/lang/go/packages.el

```diff
@@ -31,3 +33,19 @@ which require an initialization must be listed explicitly in the list.")
 
 (defun go/init-go-eldoc()
     (add-hook 'go-mode-hook 'go-eldoc-setup))
+
+(defun go/init-go-autocomplete()
+  (use-package go-autocomplete
+    :if (boundp 'ac-sources)
+  )
+)
+(defun go/init-company-go ()
+ (use-package company-go
+   :if (boundp 'company-backends)
+   :defer t
```

Comment:
This package may never load because there is no hook and only a `:config` property which is called after the package has been loaded.